dask_expr/datasets.py

```diff
@@ -221,8 +221,5 @@ def timeseries(
     if dtypes is None:
         dtypes = {"name": "string", "id": int, "x": float, "y": float}
 
-    if seed is None:
-        seed = np.random.randint(2e9)
```

Comment:
It seems like this we are somehow loosing some determinism when we remove these lines. For example, I get failures for the following test when these lines are removed:

```python
@pytest.mark.parametrize("seed", [42, None])
def test_timeseries_deterministic_head(seed):
    df = timeseries(end="2000-01-02", seed=seed)
    assert_eq(df.head(), df.head())
```